PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -1043,6 +1043,7 @@ DECLARE_SOA_EXTENDED_TABLE_USER(HfCand3ProngExt, HfCand3ProngBase, "HFCAND3PEXT"
 
 using HfCand3Prong = HfCand3ProngExt;
 using HfCand3ProngWPid = soa::Join<HfCand3Prong, HfProng0PidPi, HfProng0PidPr, HfProng0PidKa, HfProng1PidPi, HfProng1PidPr, HfProng1PidKa, HfProng2PidPi, HfProng2PidPr, HfProng2PidKa>;
+using HfCand3ProngWPidPiKa = soa::Join<HfCand3Prong, HfProng0PidPi, HfProng0PidKa, HfProng1PidPi, HfProng1PidKa, HfProng2PidPi, HfProng2PidKa>;
```

Comment:
It might be better for clarity to also rename `HfCand3ProngWPid` to `HfCand3ProngWPidPiKaPr`.

Reply:
this I can do it

Reply:
done